policyengine_us/parameters/gov/states/ri/tax/income/exemption/phase_out/percentage.yaml

```diff
@@ -0,0 +1,39 @@
+description: Rhode Island phases the exemption amount out at this rate, calculated based on the total number of exemptions and income.
+metadata:
+  type: single_amount
+  threshold_unit: /1
+  amount_unit: /1
+  label: Rhode Island exemption phase out rate 
+  reference:
+    - title: Rhode Island Resident Individual Income Tax Returm (2022 For, RI-1040) page 1&5
+      href: https://tax.ri.gov/sites/g/files/xkgbur541/files/2022-12/2022_1040WE_w_0.pdf
+    - title: Rhode Island Resident Individual Income Tax Returm (2021 For, RI-1040) page 1&5
+      href: https://tax.ri.gov/sites/g/files/xkgbur541/files/2022-01/2021_1040we_w.pdf
+    - title: Rhode Island Legal Code R.I. Gen. Laws § 44-30-2.6 (E)
```

Comment:
```suggestion
    - title: Rhode Island Legal Code R.I. Gen. Laws § 44-30-2.6 (E), (4), (b)
```

Reply:
be as accurate as possible in referencing where the number came from in the law - all yaml files